jest.setup.js

```diff
@@ -1,5 +1,5 @@
 import { configure } from 'enzyme';
-import Adapter from 'enzyme-adapter-react-16';
+import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
```

Comment:
The official enzyme adapter for react 17 isn't out yet, but this fork works for now.